docs/sorting-algorithms/quick-sort.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: Quick Sort
+slug: /sorting-algorithms/Quick-sort
+---
+
+## Introduction
+
+Quick Sort is a sorting algorithm which follows Divide and Conquer. In this algorithm it picks an element as a pivot and arrange the array according to this pivot element. The condition to arrange the array according to pivot element is that all the elements which are smaller than or equal to pivot must lies on the left side of pivot. And all the elements which are greater than pivot must lie on the right side of the pivot. Quick sort is not a in-place algorithm but to make it in-place it requires additional amounts of array.
+
+## Explanation 
+In this algorithm firstly it picks an pivot element and arrange the array according to this pivot element. The condition to arrange the array according to pivot element is that all the elements which are smaller than or equal to pivot must lies on the left side of pivot. And all the elements which are greater than pivot must lie on the right side of the pivot. After that follow same approach on left part of pivot and right part of pivot respectively until we left with single element and finally we got our array sorteed.
+### Example 1
+
+<img src="https://i.imgur.com/BEbTHiO.png" width="500" height="350"/>
+
+```
+
+ Array arr[4]:        
+ | 11 | 22 | 44 | 33 |                               
+ Pivot element=33  
+ After Partition :
+ | 11 | 22  || 33 || 44 |                                             
+ left part= | 11 | 22 | 
+ right part= | 44 |                                      
+ Pivot element= | 22 |
+ left part= | 11 |
+ | 11 || 22 || 33 | 44 |
+  Sorted!! 
```

Comment:
Can you follow a slightly more standard example? Take a look at the existing examples and see if you can format yours according to them.